flink-connectors/flink-connector-aws-kinesis-firehose/src/test/java/org/apache/flink/connector/firehose/sink/testutils/KinesisFirehoseTestUtils.java

```diff
@@ -73,4 +81,22 @@ public static void createDeliveryStream(
                 firehoseAsyncClient.createDeliveryStream(request);
         deliveryStream.get();
     }
+
+    public static DataStream<String> getSampleDataGenerator(
+            StreamExecutionEnvironment env, int endValue) {
+        ObjectMapper mapper = new ObjectMapper();
+        return env.fromSequence(1, endValue)
+                .map(Object::toString)
+                .returns(String.class)
+                .map(data -> mapper.writeValueAsString(ImmutableMap.of("data", data)));
+    }
+
+    public static List<String> getSampleDataGenerated(int endValue) throws JsonProcessingException {
```

Comment:
nit: This method name does not make sense to me, should it be `getSampleData`/`generateSampleData`/`createSampleData`

Reply:
4 hours ago intended to. I've reassessed and do not plan to implement due to lack of time.

Reply:
done.